CHANGELOG.md

```diff
@@ -3429,61 +3519,68 @@ That's why is this section formatted differently.
-[v7.0.0-beta6]: https://github.com/shopsys/shopsys/compare/v7.0.0-beta5...v7.0.0-beta6
-[v7.0.0-beta5]: https://github.com/shopsys/shopsys/compare/v7.0.0-beta4...v7.0.0-beta5
-[v7.0.0-alpha1]: https://github.com/shopsys/shopsys/compare/1.1.0...v7.0.0-alpha1
-[v7.0.0]: https://github.com/shopsys/shopsys/compare/v7.0.0-beta6...v7.0.0
-[@shopsys]: https://github.com/shopsys
-[@return]: https://github.com/return
-[@param]: https://github.com/param
 [@mposchl]: https://github.com/mposchl
-[@ShopsysFW]: https://github.com/ShopsysFW
-[@RostislavKreisinger]: https://github.com/RostislavKreisinger
-[@LukasHeinz]: https://github.com/LukasHeinz
-[6.0.0-beta21]: https://github.com/shopsys/shopsys/compare/6.0.0-beta20...6.0.0-beta21
-[6.0.0-beta20]: https://github.com/shopsys/shopsys/compare/6.0.0-beta19.2...6.0.0-beta20
-[6.0.0-beta19.2]: https://github.com/shopsys/shopsys/compare/6.0.0-beta19.1...6.0.0-beta19.2
-[6.0.0-beta19.1]: https://github.com/shopsys/shopsys/compare/6.0.0-beta19...6.0.0-beta19.1
-[6.0.0-beta19]: https://github.com/shopsys/shopsys/compare/6.0.0-beta18...6.0.0-beta19
-[6.0.0-beta18]: https://github.com/shopsys/shopsys/compare/6.0.0-beta17...6.0.0-beta18
-[6.0.0-beta17]: https://github.com/shopsys/shopsys/compare/6.0.0-beta16...6.0.0-beta17
-[6.0.0-beta16]: https://github.com/shopsys/shopsys/compare/6.0.0-beta15...6.0.0-beta16
-[6.0.0-beta15]: https://github.com/shopsys/shopsys/compare/6.0.0-alpha...6.0.0-beta15
-[6.0.0-alpha]: https://github.com/shopsys/shopsys/compare/4.0...6.0.0-alpha
-[4.0]: https://github.com/shopsys/shopsys/compare/3.1.1...4.0
-[3.1.1]: https://github.com/shopsys/shopsys/compare/3.1.0...3.1.1
-[3.1.0]: https://github.com/shopsys/shopsys/compare/0.2.0...3.1.0
-[2.3.0]: https://github.com/shopsys/shopsys/compare/0.5.0...2.3.0
-[1.1.0]: https://github.com/shopsys/shopsys/compare/1.0.1...1.1.0
-[1.0.1]: https://github.com/shopsys/shopsys/compare/1.0.0...1.0.1
-[1.0.0]: https://github.com/shopsys/shopsys/compare/0.2.1...1.0.0
-[0.6.1]: https://github.com/shopsys/shopsys/compare/0.6.0...0.6.1
-[0.6.0]: https://github.com/shopsys/shopsys/compare/0.5.1...0.6.0
-[0.5.1]: https://github.com/shopsys/shopsys/compare/0.5.0...0.5.1
-[0.5.0]: https://github.com/shopsys/shopsys/compare/0.4.0...0.5.0
-[0.4.2]: https://github.com/shopsys/shopsys/compare/0.4.1...0.4.2
-[0.4.1]: https://github.com/shopsys/shopsys/compare/0.4.0...0.4.1
-[0.4.0]: https://github.com/shopsys/shopsys/compare/0.3.0...0.4.0
-[0.3.0]: https://github.com/shopsys/shopsys/compare/0.2.0...0.3.0
-[0.2.2]: https://github.com/shopsys/shopsys/compare/0.2.1...0.2.2
-[0.2.1]: https://github.com/shopsys/shopsys/compare/0.2.0...0.2.1
-[0.2.0]: https://github.com/shopsys/shopsys/compare/0.1.0...0.2.0
-[0.1.2]: https://github.com/shopsys/shopsys/compare/0.1.1...0.1.2
-[0.1.1]: https://github.com/shopsys/shopsys/compare/0.1.0...0.1.1
-[0.1.0]: https://github.com/shopsys/shopsys/compare/6.0.0-beta21...0.1.0
```

Comment:
This is initial tag of shopsys/product-feed-google (probably) there is nothing to compare.